rankers/SimpleAggregateRanker/__init__.py

```diff
@@ -28,8 +28,7 @@ def _get_score(self, match_idx, query_chunk_meta, match_chunk_meta, *args, **kwa
         scores = match_idx[self.COL_SCORE]
         aggregated_score = self.np_aggregate_function(scores)
         if self.is_reversed_score:
-            if aggregated_score < 0:
-                raise ValueError(f'Setting is_reversed_score to "{self.is_reversed_score}" does not allow negative '
-                                 f'chunk scores (aggregated_score is {aggregated_score}).')
+            if aggregated_score == 0:
```

Comment:
what it does not allow is having the `score` to -1, right?

Reply:
sorry, I fixed it